src/hyp.jl

```diff
@@ -48,16 +50,17 @@ over_th(::Type{Float32}) = 18.714973875f0
 
 Compute hyperbolic tangent of `x`.
 """
-function tanh(x::T) where {T<:Union{Float32,Float64}}
+function tanh(x::V) where V <: FloatType
```

Comment:
Stylistically I'd prefer we left this as `where {V <: FloatType}`

https://github.com/jrevels/YASGuide 

> Type variable bindings should always be enclosed within {} brackets when using where syntax, e.g. Vector{Vector{T} where T} is good, Vector{Vector{T}} where {T} is good, Vector{Vector{T}} where T is bad.


> The return keyword should always be omitted from return statements within short-form method definitions (f(...) = ...). The return keyword should never be omitted from return statements within any other context (function ... end, macro ... end, etc.).
If a function does not have a clearly appropriate return value, then explicitly return nothing.

Reply:
Got it. Added brackets.